wordpress-mobile/WordPress-Android
# Issue/7565 -  Show icons on Stats Widget

Fixes #7565 - Stats Widget Text Misaligned

Actually, this PR resolve a problem with the icon beside the text.

After investigating several ways to solve the problem, the way in which I could make the Widget show the image was by setting a background to the ImageView instead of setting "app: srcCompat"

I'm not sure if this is the best solution, however drawable now is visible

To test:

1. Add **Today's Stats widget 5 x 1** to your home screen
2. Choose site
3. Statistics should be displayed with the respective **titles and icons on the left side of the text**
4. Notice that beside the **Blog Title** also is showing the icon. 

![screenshot_1525285457](https://user-images.githubusercontent.com/2581647/39541698-84548474-4e14-11e8-9e53-6393dbdeeee0.png)
